avocado/utils/distro.py

```diff
@@ -230,6 +266,9 @@ def get_distro(self):
 
         distro = None
 
+        if session:
+            self.session = session
```

Comment:
Unless I'm missing something, this is unnecessary if we remove the `session` argument.